bolt/test/X86/dwarf4-df-input-lowpc-ranges-cus.test

```diff
@@ -0,0 +1,97 @@
+; RUN: rm -rf %t
+; RUN: mkdir %t
+; RUN: cd %t
+;; RUN: llvm-mc -dwarf-version=4 -filetype=obj -triple x86_64-unknown-linux %p/Inputs/dwarf4-df-input-lowpc-ranges-main.s \
+; RUN: -split-dwarf-file=main.dwo -o main.o
+;; RUN: llvm-mc -dwarf-version=4 -filetype=obj -triple x86_64-unknown-linux %p/Inputs/dwarf4-df-input-lowpc-ranges-other.s \
+; RUN: -split-dwarf-file=other.dwo -o other.o
```

Comment:
I am surprised -split-dwarf-file=other.dwo works considering in assembly dwo_name is mainother.dwo

Reply:
Yes, for some reason only ran into an issue with it for the dwarf5 test. Will fix.

Reply:
If you run BOLT manually you should see:
BOLT-WARNING: Debug Fission: DWO debug information for mainOther.dwo was not retrieved and won't be updated. Please check relative path.